### ISSUE TITLE
Outdated miIO Device Library

### ISSUE THREAD

**Issue by Dark-Noir** [aformusatii/node-red-contrib-xiaomi-miio#7]
The "miIO Device Library" is outdated and does not supports newer devices.

You should switch to the "MiJia IO Device Library": https://github.com/salamwaddah/mijia-io.

**Comment by spongioblast** [comment]
Yes could you pretty please upgrade it? I d love to add support for the Air Purifier Pro H, realistically this is only possible with the current library. Thanks!
